app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt

```diff
@@ -78,6 +81,34 @@ class ParseSusiResponseHelper {
                 Timber.e(e)
             }
 
+            Constant.TABLE -> try {
+                count = 0
+                val listColumn: ArrayList<String> = ArrayList()
+                val listColVal: ArrayList<String> = ArrayList()
+                val listTableData: ArrayList<String> = ArrayList()
+
+                for (tableanswer in susiResponse.answers) {
+                    for (answer in tableanswer.actions) {
+                        val map = answer.columns
+                        map?.forEach {
```

Comment:
Replace with answer.columns?.forEach

No need to create variables which are unused